### ISSUE TITLE
Open sourcing Flux Studio

### ISSUE THREAD

**Issue by alesrebec** [flux3dp/flux-studio#13]
Could you open source the latest versions of Flux Studio app so the people who bougth it and supported the project from day 1 can create new releases?

/cc @simonxeko @net5566

**Comment by kintaro1981** [comment]
I agree, right know if I'm not wrong from the website is possibile to download 1.0.0 beta1.
On github the last release is 0.7.4
